pkg/api/getambassador.io/v3alpha1/crd_host.go

```diff
@@ -103,15 +103,7 @@ type HostSpec struct {
 	// certificates.  If ACME is enabled (see $acmeProvider), then the
 	// default is $hostname; otherwise the default is "".  If the value
 	// is "", then we do not do TLS for this Host.
-	//
-	// Note that this is a native-Kubernetes-style core.v1.LocalObjectReference, not
-	// an Ambassador-style `{name}.{namespace}` string.  Because we're opinionated, it
-	// does not support referencing a Secret in another namespace (because most native
-	// Kubernetes resources don't support that), but if we ever abandon that opinion
-	// and decide to support non-local references it, it would be by adding a
-	// `namespace:` field by changing it from a core.v1.LocalObjectReference to a
-	// core.v1.SecretReference, not by adopting the `{name}.{namespace}` notation.
```

Comment:
lol @ how relevant this comment was

But also perhaps it's worth keeping the note about it not being a `{name}.{namespace}` string, for people migrating to this from TLSContexts.  Something like "Note that this is a native-Kubernetes-style `core.v1.SecretReference`, not an old-Emissary-style `{name}.{namespace}` string."

Reply:
That's a good idea for sure and something to call out when I make a docs update for this for sure.

Reply:
Ideally the `kubectl describe` docs are all helpful, and One Day™ we'll have website docs generated from it.